x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality_panel/data_quality_details/indices_details/pattern/index.tsx

```diff
@@ -320,6 +358,7 @@ const PatternComponent: React.FC<Props> = ({
                   onCheckNowAction={handleFlyoutCheckNowAndExpandAction}
                   onViewHistoryAction={handleFlyoutViewCheckHistoryAction}
                   sorting={sorting}
+                  firstIndexName={items[0]?.indexName}
```

Comment:
I see we are already passing `items`, do we really need this new prop? Could we just do `const firstIndexName = items[0]?.indexName` inside the _SummaryTable_ component?

Reply:
absolutely! good catch, thanks